uwblueprint/planet-read
# Resize translation progress font with font scaler

## Notion ticket link



[Resize translation progress font with font scaler](https://www.notion.so/uwblueprintexecs/Resize-translation-progress-font-with-font-scaler-8db69cadec194da89c2a1f9365b0eacc)



## Implementation description

- Pass `fontSize` (which is changed by the font scaler) to the `ProgressBar` component
- Use a switch statement to determine the appropriate size of progress bar



## Steps to test

1. Login as Carl
2. Go to Great Gatsby translation
3. Resize text (move slider both up and down), and verify that progress bar resizes properly
4. Send Great Gatsby for review
5. Login as Dwight
6. Go to Great Gatsby translation
7. Resize text (move slider both up and down), and verify that both progress bars resize properly
8. Go to profile
9. Add a new language or level up a language (i.e. make a new test)
10. Send the test for review
11. Login as Angela
12. Go to grade test
13. Resize text (move slider both up and down), and verify that progress bar resizes properly



## What should reviewers focus on?

- Does it work?
- Does the sizing and styling look ok?
- Does the implementation (please see implementation description) make sense?
    - Would it be better to pass an integer (e.g. 0, 1, 2) instead of `fontSize`(i.e. "12px", "16px", "24px") to the `ProgressBar` component?

## Checklist
